### PR TITLE
nav menu open by default on home #848

### LINKED ISSUE #848: Navigation menu open by default on homepage
**Description**:
As part of https://github.com/ral-facilities/datagateway/issues/954, the navigation drawer should be open on the datagateway homepage by default.

**Acceptance criteria**:
- [ ] Menu opens by default on the homepage

### PR DESCRIPTION
**Pull request by joshuadkitenge**
## Description
As part of ral-facilities/datagateway#954, the navigation drawer should be open on the datagateway homepage by default.

## Testing instructions
Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking
closes #848

### CODE REVIEW

**Comment by joshuadkitenge** [comment]
default-open

File: cypress/integration/login.spec.ts [hunk 1]
```diff
@@ -105,7 +105,7 @@ describe('Login', () => {
       (window) =>
         expect(window.localStorage.getItem('scigateway:token')).not.be.null
     );
-    cy.get('button[aria-label="Open navigation menu"]').should('be.visible');
+    cy.get('[aria-label="Close navigation menu"]').should('be.visible');
```
Comment (joelvdavies) [inline cypress/integration/login.spec.ts:108]: Could this be change to have 'button' like the others?

**Review by joelvdavies — CHANGES_REQUESTED** [review]
Looks good, just a small change.

**Review by joelvdavies — APPROVED** [review]
CI passes, comment resolved.

**Pull request merged**